config.json

```diff
@@ -73,5 +73,15 @@
     "current": {
       "1.0": "1.0.3"
     }
+  },
+  "vip-governance": {
+    "repo": "https://github.com/Automattic/vip-governance",
+    "folderPrefix": "vip-integrations/vip-governance-",
+    "lowestVersion": "1.0",
+    "skip": [],
+    "ignore": [],
+    "current": {
```

Comment:
Not a big deal, but you can leave `current: {}` empty and it'll get filled in automatically when it's picked up. We did the [same thing](https://github.com/Automattic/vip-go-mu-plugins-ext/pull/35/files#diff-587cb980af76fdc7e52369fd0b9d926dff266976b6f8ac631e358fecc49ff8cfR54) for the Block Data API.

Reply:
Fixed